src/components/bookmarksList/bookmarksList.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import AccountVisual from '../accountVisual';
+import EmptyState from '../emptyStateV2';
+import Box from '../boxV2';
+import { tokenMap } from '../../constants/tokens';
+import regex from '../../utils/regex';
+import svg from '../../utils/svgIcons';
+import { SecondaryButtonV2 } from '../toolbox/buttons/button';
+import routes from '../../constants/routes';
+import styles from './bookmarksList.css';
+
+class BookmarksList extends React.Component {
+  onBookmarkSelected(address) {
+    const { history } = this.props;
+    history.push(`${routes.explorer.path}/accounts/${address}`);
+  }
+
+  getBookmarkListBasedOnSelectedToken() {
+    const { bookmarks, token } = this.props;
+
+    const actualBookmarks = { ...bookmarks };
+    return actualBookmarks[token.active].length > 5
+      ? actualBookmarks[token.active].slice(0, 5)
+      : bookmarks[token.active];
+  }
+
+  displayAddressBasedOnSelectedToken(address) {
+    const { token } = this.props;
+
+    return token.active === tokenMap.LSK.key
+      ? address
+      : address.replace(regex.btcAddressTrunk, '$1...$3');
+  }
+
+  render() {
+    const { t, token } = this.props;
+
+    const selectedBookmarks = this.getBookmarkListBasedOnSelectedToken();
+
+    return (
+      <Box className={` ${styles.box} bookmarks-list`}>
+        <header>
+          <h2>Bookmarks</h2>
+        </header>
+        <div className={`${styles.bookmarkList} bookmark-list-container`}>
+        {
+          selectedBookmarks.length
+          ? selectedBookmarks.map(bookmark =>
+            <div
+              key={bookmark.address}
+              className={`${styles.row} bookmark-list-row`}
+              onClick={() => this.onBookmarkSelected(bookmark.address)}>
```

Comment:
Isn't it possible to instead of having the `onClick`, to do a history.push, wrap this inside a `<Link>`?